lib/exposes.js

```diff
@@ -479,6 +479,9 @@ module.exports = {
         light_brightness_colortemp_colorxy: (colorTempRange) => new Light().withBrightness().withColorTemp(colorTempRange).withColorTempStartup(colorTempRange).withColor(['xy']),
         light_brightness_colorxy: () => new Light().withBrightness().withColor((['xy'])),
         light_colorhs: () => new Light().withColor(['hs']),
+        light_color_options: () => new Composite('light_color_options', 'light_color_options')
+            .withDescription('Configure lightingColorCtrl options attribute.')
```

Comment:
Following sounds more friendly in my opinion: 
- `Configure lightingColorCtrl options attribute.` ->  `Configure lighting cluster options attribute.`
- `Execute changes when onOff cluster set to off` -> `Execute color(temperature) changes even when light is off`